packages/commonwealth/client/styles/sublayout_header_left.scss

```diff
@@ -3,10 +3,30 @@
 .SublayoutHeaderLeft {
   align-items: center;
   display: flex;
+  gap: 8px;
   flex-direction: row;
+  height: $global-header-height;
 
-  .CommunityAvatar {
-    margin-right: 8px;
+  .Icon.commonLogo {
+    cursor: pointer;
```

Comment:
If you make it a `CWIconButton` as suggested above, you won't need `cursor: pointer`.